CHANGELOG.md

```diff
@@ -10,6 +10,18 @@
 
 ------------
```

Comment:
(I would entirely remove that file ... maintaining CHANGELOG.md for every app is not a thing, and it contains only stuff that make sense for packagers)

Reply:
To me, the file is for "packagers" or "advancers users" who would like to know what is inside the new version released.

Is there "any other way" to "tell or teach" packagers about this? I see two important stuff to document:
1. "new features / changes" in the packages (changelog as fir now)
2. what is important to know about the app (not the YunoHost packaging" itself which is in the main YNH doc